pubspec.yaml

```diff
@@ -10,15 +10,33 @@ environment:
   sdk: '>=2.3.0 <3.0.0'
 
 dependencies:
-  args: '>=0.13.7 <2.0.0'
+  args: ^1.5.0
+  checked_yaml: ^1.0.0
+  glob: ^1.2.0
+  json_annotation: ^3.0.0
   logging: ^0.11.3+1
   path: ^1.4.2
   pub_semver: ^1.4.1
   yaml: ^2.1.13
 
 dev_dependencies:
+  build_runner: ^1.7.3
+  json_serializable: ^3.2.5
   test: ^1.9.4
   test_descriptor: ^1.2.0
+  workiva_analysis_options: ^1.0.0
 
 executables:
   dependency_validator:
+
+dependency_validator:
+  exclude:
+    - 'test_fixtures/**'
+  ignore:
+    - json_serializable
+    - _foo
+    - foo
+    - _bar
+    - bar
+    - foo1
+    - foo_foo
```

Comment:
note to self: these packages only have to be ignored because our regex approach is picking up on imports in strings and not real imports. We should consider using the `analyzer` package and an `AstVisitor` to detect imports and exports in Dart code; it should still be relatively fast since it only needs to parse the AST and not analyze the files.